python/pyspark/sql/streaming.py

```diff
@@ -504,105 +504,13 @@ def json(self, path, schema=None, primitivesAsString=None, prefersDecimal=None,
         path : str
             string represents path to the JSON dataset,
             or RDD of Strings storing JSON objects.
-        schema : :class:`pyspark.sql.types.StructType` or str, optional
```

Comment:
this doc doesn't exist anymore too. we should keep it.

Reply:
I added it to Data Source Options table!

<img width="806" alt="Screen Shot 2021-05-13 at 1 30 11 PM" src="https://user-images.githubusercontent.com/44108233/118077601-62bc5f00-b3ef-11eb-9350-c62b370e167c.png">